spec/system/consumer/split_checkout_tax_not_incl_spec.rb

```diff
@@ -257,13 +261,13 @@
 
 private
 
-def assert_db_tax
+def assert_db_tax_added
```

Comment:
Interesting. I would have never thought of that. But now looking at the two files, they have the same top-level description: "As a consumer, I want to see adjustment breakdown"

I guess that rspec creates a class derived from this string and if you re-use the same description then these specs get merged together, as if they were in the same file.